### ISSUE TITLE
How was Hakiri integrated with this project?

### ISSUE THREAD

**Issue by nasifimtiazohi** [18F/micropurchase#1532]
I see from the readme badge that this project is integrated with Hakiri.

However, from hakiri log, https://hakiri.io/github/18F/micropurchase/master, I only see a handful commits, and those commits are not consecutive either.  Seems like the hakiri log contains commits in a scattered order.

Can you tell me why is this and how exactly was hakiri used with this project?

@jessieay @stvnrlly

**Comment by jessieay** [comment]
Unfortunately, I don't know much about how Hakiri works. This project might , have been my only experience with Hakiri. At one point I seem to remember having some Hakiri webhooks on PRs so you couldn't merge if there were any issue detected. It is possible that was turned off, which is why there are only a few commits on hakiri.io.

If you look at the README, it says that we are using Hakiri to monitor for any known issues with Ruby and Rails versions and says to run command locally to check for security issues: https://github.com/18F/micropurchase#security-scans

So perhaps we just included it to be used as a local utility? 

Wish I remembered more!
